### ISSUE TITLE
Await for the gecko profiler start instead of adding a synthetic delay

### ISSUE THREAD

**Issue by canova** [sitespeedio/browsertime#1857]
### Feature/improvement

Currently browsertime adds a delay after the gecko profiler start command:
https://github.com/sitespeedio/browsertime/blob/79e9b0e1f72c3005641b1131173cdaaf24f93a51/lib/firefox/geckoProfiler.js#L89-L90

**Comment by soulgalore** [comment]
Hi @canova ah cool, so then I can just remove the extra wait and it will work as it should right?

**Comment by canova** [comment]
Yes, it should work as expected without the extra wait if we properly await the `Services.profiler.StartProfiler`.
Not sure if we are doing it already? [I see an `await`](https://github.com/sitespeedio/browsertime/blob/79e9b0e1f72c3005641b1131173cdaaf24f93a51/lib/firefox/geckoProfiler.js#L89) in front of the `runner.runPrivilegedScript` call but there is no await inside the script that's passed to `runPrivilegedScript` as an argument. Probably it's not awaiting for that one?
